qa/os/src/test/java/org/elasticsearch/packaging/test/ArchiveTests.java

```diff
@@ -47,9 +53,32 @@ public static void filterDistros() {
         assumeTrue("only archives", distribution.isArchive());
     }
 
+    private static String superuser = "test_superuser";
+    private static String superuserPassword = "test_superuser";
+
     public void test10Install() throws Exception {
         installation = installArchive(sh, distribution());
         verifyArchiveInstallation(installation, distribution());
+        Result result = sh.run(
+            installation.executables().usersTool + " useradd " + superuser + " -p " + superuserPassword + " -r " + "superuser"
+        );
+        assumeTrue(result.isSuccess());
+        // See https://bugs.openjdk.java.net/browse/JDK-8267701. In short, when generating PKCS#12 keystores in JDK 12 and later
```

Comment:
>  Is there any drawback to using these JDK 11 compatible settings by default?  

This is not configurable in Java when creating the keystore so we'd have to ship with a custom `java.security` file in our bundled JDK and I don't think we should do that. 

> I wonder if this is going to be a problem for folks that initially run auto-configuration with the bundled JDK and then subsequently switch to another JDK.

Yes this would be but since 

> we intend to move to JDK 17 as the minimum

I consider this a non-issue as this wouldn't affect anyone in the next major ES version. I just wanted a solution for until we do the bump to JDK17 here